### PR TITLE
chore: fixed broken urls and reported typo

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR fixes two identified broken URLs and a typo in the Helm Install page.

## Review Changes

💻 [Preview](https://deploy-preview-1244--docs-spectrocloud.netlify.app/kubernetes-knowlege-hub/how-to/how-to-retrieve-images-from-private-registry)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1244#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@karl-cardenas-coding: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 21f1a6d840373be4be5d4f9b2f607eacb4cceb8b | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1244/librarium_build/1647972891368624128) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/19-kubernetes-knowlege-hub/00-how-to/02-how-to-retrieve-images-from-private-registry.md
Comment (ritawatson) [inline content/docs/19-kubernetes-knowlege-hub/00-how-to/02-how-to-retrieve-images-from-private-registry.md:35]: @karl-cardenas-coding Wouldn't we keep Clusters as the link text initial capped because it's the index page? And as a doc, it's "Clusters"?

**Review by ritawatson — APPROVED** [review]
LGTM with one question about initial capping "Clusters" as the link text.

**Review by ritawatson — APPROVED** [review]
Just one suggestion to keep Clusters link text capped. Otherwise, LGTM.

**Pull request merged**
